tracking_analyzer/admin.py

```diff
@@ -67,10 +68,10 @@ def ip_address_link(self, obj):
         Define the 'IP Address' column rows display.
         """
         if obj.ip_address:
-            return '<a href="{0}?ip_address__exact={1}">{1}</a>'.format(
+            return format_html('<a href="{0}?ip_address__exact={1}">{1}</a>'.format(
```

Comment:
This line is not PEP8 compliant. Could you please fix it? Then it will pass the [Travis build](https://travis-ci.org/github/jose-lpa/django-tracking-analyzer/jobs/662243778)

```
return format_html(
    '<a href="{0}?ip_address__exact={1}">{1}</a>'.format(
        reverse('admin:tracking_analyzer_tracker_changelist'),
        obj.ip_address
    )
)
```

Thank you!